### PR TITLE
More work on 1.13

### PR DESCRIPTION
**Pull request by Karang**
Work in progress

### CODE REVIEW

**Comment by rom1504** [comment]
could you please rebase on 1.13 branch ?

File: src/lib/plugins/placeBlock.js [hunk 1]
```diff
@@ -81,43 +93,22 @@ module.exports.player = function (player, serv, { version }) {
...
+    serv.playSound(sound, player.world, placedPosition.offset(0.5, 0.5, 0.5), {
+      pitch: 0.8
     })
+
+    if (player.gameMode === 0) { player.inventory.slots[36 + player.heldItemSlot]-- }
+
+    const stateId = serv.supportFeature('theFlattening') ? (blocks[id].minStateId + data) : (id << 4 | data)
+    player.setBlock(placedPosition, stateId)
+
+    if (id === 63 || id === 68) {
```
Comment (rom1504) [inline src/lib/plugins/placeBlock.js:107]: this seems wrong (hardcoded ids)

Comment (Karang) [reply src/lib/plugins/placeBlock.js:107]: indeed, this part have nothing to do here anyway, it should be move in signs.js

Comment (Karang) [reply src/lib/plugins/placeBlock.js]: done

**Pull request merged**